FastSimulation/Muons/plugins/FastTSGFromL2Muon.cc

```diff
@@ -59,27 +41,25 @@ void FastTSGFromL2Muon::produce(edm::Event& ev, const edm::EventSetup& es) {
   theRegionBuilder->setEvent(ev, es);
 
   // Retrieve the Monte Carlo truth (SimTracks)
-  edm::Handle<edm::SimTrackContainer> theSimTracks;
-  ev.getByLabel(theSimTrackCollectionLabel, theSimTracks);
+  const edm::Handle<edm::SimTrackContainer>& theSimTracks = ev.getHandle(simTrackToken_);
 
   // Retrieve L2 muon collection
-  edm::Handle<reco::TrackCollection> l2muonH;
-  ev.getByLabel(theL2CollectionLabel, l2muonH);
+  const edm::Handle<reco::TrackCollection>& l2muonH = ev.getHandle(l2TrackToken_);
 
   // Retrieve Seed collection
   unsigned seedCollections = theSeedCollectionLabels.size();
   std::vector<edm::Handle<edm::View<TrajectorySeed> > > theSeeds;
   theSeeds.resize(seedCollections);
   unsigned seed_size = 0;
   for (unsigned iseed = 0; iseed < seedCollections; ++iseed) {
-    ev.getByLabel(theSeedCollectionLabels[iseed], theSeeds[iseed]);
+    ev.getByToken(seedToken_[iseed], theSeeds[iseed]);
     seed_size += theSeeds[iseed]->size();
   }
 
   // Loop on L2 muons
   unsigned int imu = 0;
   unsigned int imuMax = l2muonH->size();
-  // std::cout << "Found " << imuMax << " L2 muons" << std::endl;
+  edm::LogVerbatim("FastSim") << "Found " << imuMax << " L2 muons";
```

Comment:
@bsunanda , the same concern as to other PRs: class name should be added

Reply:
Taking care of it